Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -79,7 +79,11 @@ class SearchPeopleTableViewCell: UITableViewCell {
     
     func configure(for user: User, index: Int) {
         self.index = index
-        profilePictureImageView.image = #imageLiteral(resourceName: "sample_profile_pic")
+        if let url = user.imageURL, let data = try? Data(contentsOf: url) {
```

Comment:
profilePictureImageView to be ImageView then call setLoadImageUrlAsync(url:...) or wtvr the function is called.